d2l-table-style.js

```diff
@@ -1,45 +1,33 @@
+import '@brightspace-ui/core/components/colors/colors.js';
 import '@polymer/polymer/polymer-legacy.js';
-import 'd2l-colors/d2l-colors.js';
-import './d2l-table-shared-styles.js';
+import { tableStyles } from '@brightspace-ui/core/components/table/table-wrapper.js';
+
+if (document.getElementById('d2l-table-style') === null) {
+	const style = document.createElement('style');
+	style.id = 'd2l-table-style';
+	style.appendChild(document.createTextNode(tableStyles.cssText));
+	document.head.appendChild(style);
```

Comment:
This code is a similar approach to the legacy Polymer style module below that previously contained most of these styles, but this works with the Lit shared styles. BSI is going to have code identical to this (sharing the same `id`) so that the style will be present for monolith pages that use `<d2l-table-wrapper>` + `<table class="d2l-table">`.

Reply:
Does this `id` clash with the module id below?  I guess not if it's working and both get added properly

Reply:
Hmm, that's a good point. I'm not noticing any issues but they do have the same `id` so I think I'll change it here and in BSI just to be extra safe -- thanks!